clang-tools-extra/test/clang-tidy/checkers/cppcoreguidelines/missing-std-forward.cpp

```diff
@@ -147,4 +147,24 @@ class AClass {
   T data;
 };
 
+template <class T>
+void lambda_value_reference(T&& t) {
+  [&]() { T other = std::forward<T>(t); };
+}
+
+template<typename T>
+void lambda_value_reference_capture_list_ref_1(T&& t) {
+    [=, &t] { T other = std::forward<T>(t); };
+}
+
+template<typename T>
+void lambda_value_reference_capture_list_ref_2(T&& t) {
+    [&t] { T other = std::forward<T>(t); };
+}
+
+template<typename T>
+void lambda_value_reference_capture_list(T&& t) {
+    [t = std::forward<T>(t)] { t(); };
+}
+
```

Comment:
missing:
```
template <class T>
void lambda_value_reference(T&& t) {
  [&x = t]() { T other = std::forward<T>(x); };
}
```

and same with std::forward.

Reply:
Fixed.